test/cutting/cut_finding/test_cco_utils.py

```diff
@@ -0,0 +1,134 @@
+from __future__ import annotations
+
+import pytest
+from pytest import fixture
+from qiskit.circuit.library import EfficientSU2
+from qiskit import QuantumCircuit, QuantumRegister
+from qiskit.circuit import Qubit, Instruction, CircuitInstruction
+from circuit_knitting.cutting.cut_finding.cco_utils import (
+    qc_to_cco_circuit,
+    cco_to_qc_circuit,
+)
+from circuit_knitting.cutting.cut_finding.circuit_interface import (
+    SimpleGateList,
+    CircuitElement,
+)
+
+# test circuit 1.
+tc_1 = QuantumCircuit(2)
+tc_1.h(1)
+tc_1.barrier(1)
+tc_1.s(0)
+tc_1.barrier()
+tc_1.cx(1, 0)
+
+# test circuit 2
+tc_2 = EfficientSU2(2, entanglement="linear", reps=2).decompose()
+tc_2.assign_parameters([0.4] * len(tc_2.parameters), inplace=True)
```

Comment:
Ideally these would be in fixtures so that they are executed only if a test is run where the are needed.  The way it currently is, you can't import this test module without running these lines.

Reply:
Now I'm starting to suspect that this ended up that way because it was unclear how to reconcile my suggestion with `pytest.parametrize`.  Is that right?  If so, we can punt on this.

Reply:
https://github.com/Qiskit-Extensions/circuit-knitting-toolbox/blob/main/test/cutting/test_cutting_roundtrip.py

pytest fixture example, for reference, if it's useful

Reply:
Yeah so the reason for not using fixtures in that test was that `qc_to_cco_circuit` needs to be inputted a `QuantumCircuit`, rather than a function that generates one. Using it with `pytest.mark.parametrize` would have meant (I think) calling that fixture, storing the result, and then passing that variable to `pytest.mark.parametrize`. To avoid that, I decided to do it this way. Does that make sense?

Reply:
I opened #521 as a suggestion.